### ISSUE TITLE
местоположение "кнопка" - СТОП

### ISSUE THREAD

**Issue by Janna-Janna** [SBProX/SBProX-Dev#1928]
**Детальное описание**  КОГДА СОСРЕДОТОЧЕН НА ГРАФИКЕ И ЛИСТАЕШЬ +1TF, ЧАСТО СЛУЧАЙНО НАЖИМАЕТСЯ КНОПКА "СТОП", И ЭТО ПЕЧАЛЬНО, ВЕДЬ ПРИХОДИТСЯ СНОВА ЖДАТЬ ПЕРЕЗАГРУЗКУ ГРАФИКА

**Скриншоты**
![3 ПРОБЛЫ](https://user-images.githubusercontent.com/63582167/112749903-2a032a80-8fe7-11eb-8ec4-06b1e7ebb82d.png)

**Comment by ProgramGun** [comment]
Откат графика назад при тестах, как бы хорошая штука. Но чисто психологически даже на тестах, когда вы их делаете много - это может привести к "жульничеству" и некорректным результатам в итоге.... Имхо, это некий соблазн, психология так работает.

**Comment by ProgramGun** [comment]
Про кнопку стоп согласен - также часто сталкиваюсь. Нужны хоткеи.
И про инструменты тоже 100%, чтобы прилинковать линии к графику приходится разворачивать и т.д.

**Comment by Janna-Janna** [comment]
Добрый день.
Стоит поработать над психологией.
  
>Среда, 31 марта 2021, 2:39 +03:00 от ProgramGun ***@***.***>:
> 
> 
>Откат графика назад при тестах, как бы хорошая штука. Но чисто психологически даже на тестах, когда вы их делаете много - это может привести к "жульничеству" и некорректным результатам в итоге.... Имхо, это некий соблазн, психология так работает.
>—
>You are receiving this because you authored the thread.
>Reply to this email directly,  view it on GitHub , or  unsubscribe . 
 
 
С уважением Жанна Мин.

**Comment by Vponed** [comment]
> ат графика назад при тестах, как бы хорошая штука. Но чисто психологически даже на тестах, когда вы их делаете много - это может привести к "жульничеству" и некорректным результатам в итоге.... Имхо, это некий соблазн, психология так работает. —
хороший способ над ней поработать. -1тф очень не хватает.

**Comment by jekksnn** [comment]
Согласен, кнопки СТОП на основном графике вообще не должно быть. Пусть остается в настройках в отдельном окне, когда нужно вызвал Симулятор и остановил. а то рука дрогнула и начинай все сначала
